pyaedt/modeler/cad/Primitives.py

```diff
@@ -2735,28 +2738,34 @@ def _check_material(self, matname, defaultmatname):
             Name of the material.
         defaultmatname : str
             Name of the default material to assign if ``metname`` does not exist.
+        threshold : float
+            Threshold conductivity to distinguish dielectric from conductor.
+            Default value is 100000 S/m.
```

Comment:
```suggestion
            The default value is ``100000``.
```
the units should be part of the description since they are not part of the float value.

Reply:
Units for conductivity are "Siemens per meter." It is important that this be documented.

Reply:
ah ok. I understand.